fe/fe-core/src/main/java/org/apache/doris/binlog/DropTableRecord.java

```diff
@@ -39,7 +41,10 @@ public DropTableRecord(long commitSeq, DropInfo info) {
         this.dbId = info.getDbId();
         this.tableId = info.getTableId();
         this.tableName = info.getTableName();
-        this.rawSql = String.format("DROP TABLE IF EXISTS `%s`", this.tableName);
+        this.isView = info.isView();
+        this.rawSql = info.isView()
+            ? String.format("DROP VIEW IF EXISTS `%s`", this.tableName)
+            : String.format("DROP TABLE IF EXISTS `%s`", this.tableName);
```

Comment:
It seems that apache doris hasn't DROP VIEW statement.

Reply:
![image](https://github.com/user-attachments/assets/a46323a6-441a-4c3f-8f63-7f78a9daa28b)